### ISSUE TITLE
Crashes when loading mods on Linux using yuzu-mainline-git AUR package

### ISSUE THREAD

**Issue by anzix** [amakvana/YuzuModDownloader#34]
**Describe the bug**
When you click the button to download mods from any server (Official or The Boy 181), YuzuModDownloader simply crashes leaving the downloaded xml files. No mods have been downloaded to the `~/.local/share/yuzu/load/<ID>/` folder

**To Reproduce**
1. Build the [yuzu-mainline-git](https://aur.archlinux.org/packages/yuzu-mainline-git) package from AUR
2. Copying the YuzuModDownloader Linux binary to the Desktop
3. Making it executable
4. From the terminal, I run inside the Desktop folder with the command `./YuzuModDownloader` and downloading from any server (Official or TheBoy181)
4. YuzuModDownloader crashes leaving the downloaded xml files

**Expected behavior**
I was expecting to download mods without crash

**Screenshots**
None

**Desktop/Laptop (please complete the following information):**
 - OS: Arch Linux, KDE Plasma 5.27.10, X11 Session, Kernel 6.6.8-zen1-1-zen
 - Yuzu Version on old 1644.r0.gd2b1ae6-1 and new 1662.r0.g6979cd3-1
 - Yuzu Installed
 - Game Title: None
 - Game Title ID: None

**Additional context**
Here is the program log

```
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '/home/user/Desktop/"/home/user/.local/share/yuzu/load"'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String, Boolean )
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String, Boolean, EnumerationOptions )
   at System.IO.Enumeration.FileSystemEnumerable`1.DelegateEnumerator..ctor(FileSystemEnumerable`1, Boolean)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String, FindTransform, EnumerationOptions, Boolean)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String, FindTransform , EnumerationOptions )
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserDirectories(String, String, EnumerationOptions)
   at System.IO.Directory.InternalEnumeratePaths(String, String, SearchTarget, EnumerationOptions)
   at System.IO.Directory.GetDirectories(String, String, EnumerationOptions)
   at System.IO.Directory.GetDirectories(String)
   at YuzuModDownloader.Classes.Downloaders.ModDownloader.DeleteDownloadedModArchiveFiles()
   at YuzuModDownloader.Classes.Downloaders.ModDownloader.DownloadModsAsync(List`1)
   at YuzuModDownloader.Classes.Downloaders.OfficialYuzuModDownloader.DownloadModsAsync(List`1)
   at YuzuModDownloader.MainWindow.BtnDownload_ClickAsync(Object, RoutedEventArgs)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Avalonia.Threading.SendOrPostCallbackDispatcherOperation.InvokeCore() in /_/src/Avalonia.Base/Threading/DispatcherOperation.cs:line 392
   at Avalonia.Threading.DispatcherOperation.Execute() in /_/src/Avalonia.Base/Threading/DispatcherOperation.cs:line 262
   at Avalonia.Threading.Dispatcher.ExecuteJob(DispatcherOperation) in /_/src/Avalonia.Base/Threading/Dispatcher.Queue.cs:line 117
   at Avalonia.Threading.Dispatcher.ExecuteJobsCore() in /_/src/Avalonia.Base/Threading/Dispatcher.Queue.cs:line 148
   at Avalonia.Threading.Dispatcher.Signaled() in /_/src/Avalonia.Base/Threading/Dispatcher.Queue.cs:line 128
   at Avalonia.X11.X11PlatformThreading.CheckSignaled() in /_/src/Avalonia.X11/X11PlatformThreading.cs:line 119
   at Avalonia.X11.X11PlatformThreading.RunLoop(CancellationToken) in /_/src/Avalonia.X11/X11PlatformThreading.cs:line 192
   at Avalonia.Threading.DispatcherFrame.Run(IControlledDispatcherImpl) in /_/src/Avalonia.Base/Threading/DispatcherFrame.cs:line 117
   at Avalonia.Threading.Dispatcher.PushFrame(DispatcherFrame) in /_/src/Avalonia.Base/Threading/Dispatcher.MainLoop.cs:line 53
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken) in /_/src/Avalonia.Base/Threading/Dispatcher.MainLoop.cs:line 80
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[]) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 122
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder, String[], ShutdownMode ) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 215
   at YuzuModDownloader.Program.Main(String[])
[1]    3758985 IOT instruction (core dumped)  ./YuzuModDownloader
```

**Comment by amakvana** [comment]
Apologies for the delayed response! Thanks for opening up the issue! 

This is due to a error in the parsing of the paths, it looks like Yuzu now wraps paths with `"`, which at the time wasn't accounted for within my path parsing. 

Next update will fix this

**Comment by garbear** [comment]
Thanks for looking into this @amakvana! I hit the same problem, and reinitializing my `qt-config.ini` gave me `load_directory` fields without `"`, and now the doanloader is working correctly again.

**Comment by amakvana** [comment]
@garbear @anzix 

Hi both! 

Below is a Linux test build - fixing all repo's, path scraping for Linux (for Flatpak and deb installers) and some QOL UI tweaks. Please test drive and see if the issues you both were having, have been resolved.

Same instructions to run it as always:
```
cd ~/Desktop
chmod +x YuzuModDownloader
./YuzuModDownloader
```
Cheers! 

[YuzuModDownloader_1.4.1.0_Linux_Test_Build.zip](https://github.com/amakvana/YuzuModDownloader/files/14099865/YuzuModDownloader_1.4.1.0_Linux_Test_Build.zip)

**Comment by amakvana** [comment]
Closed - Fixed in v1.4.1.0
https://github.com/amakvana/YuzuModDownloader/releases/latest
